### PR TITLE
Use any available biometric

### LINKED ISSUE #514: Further technical details in upcoming SP 800 documentation
### All Fields Are Required
**Organization Name (N/A, if individual)**: Secure Technology Alliance

**Organization Type (see below for codes)**: 2 - Industry
 
**Reference (Include section/paragraph or pdf line number)**: 2.5 Line 631-633
 
**Comment (Include rationale for comment)**: "With latest NIST facial recognition test surpassing  both Iris and finger, it should be an alternate not secondary"

**Suggested Change**: The electronic facial image is a alternate means of authentication during operator-attended PIV issuance and maintenance processes.  Further technical details in upcoming SP 800 documentation

---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

### PR DESCRIPTION
**Pull request by gfiumara**
Closes #514.

### CODE REVIEW

**Review by jricher — APPROVED** [review]

File: _FIPS201/requirements.md [hunk 1]
```diff
@@ -130,16 +130,14 @@ Agencies **MAY**
...
+Electronic iris images are an additional means of authentication during PIV issuance and maintenance processes.
 
 The electronic facial image **SHALL** be stored on the PIV Card as described in [Section 4.2.3.1](frontend.md#s-4-2-3-1).
 It **SHALL** be printed on the PIV Card according to [Section 4.1.4.1](frontend.md#s-4-1-4-1). 
 The image **MAY** be used for cardholder authentication (BIO or BIO-A) as described in [Section 6.2.1](authentication.md#s-6-2-1).
 It **MAY** be retrieved and displayed on guard workstations to augment 
 other authentication processes from [Section 6.2](authentication.md#s-6-2).
-The electronic facial image is a secondary means of authentication during operator-attended PIV issuance and maintenance processes
-when fingerprint biometric data records are unavailable.
+The electronic facial image is an additional means of authentication during operator-attended PIV issuance and maintenance processes.
```
[the remark this answers is not in the document]
Comment (gfiumara) [reply _FIPS201/requirements.md:140]: While I can't answer that question, if the required error rates for all 3 modalities in a future NIST SP 800-76-3 are all equivalent, I don't see a reason why we should prefer one over the other.

**Review by jimfenton — COMMENTED** [review]
Same question as Andy on the operator-attended requirement for facial image.

File: _FIPS201/requirements.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jricher) [reply _FIPS201/requirements.md:140]:
```suggestion
The electronic facial image is an additional means of authentication during PIV issuance and maintenance processes.
```

File: _FIPS201/requirements.md [hunk 2]
```diff
@@ -332,9 +330,8 @@ of the federal department or agency.
 + During the issuance process, the issuer **SHALL** verify that the individual to whom the PIV Card is to be
     issued is the same as the intended applicant/recipient as approved by the appropriate authority.
     Before the PIV Card is provided to the applicant, the issuer **SHALL** perform a one-to-one comparison of the
-    applicant against biometric data records available on the PIV Card or in the PIV enrollment record. The one-to-one
-    comparison requires either a comparison of fingerprints or, if unavailable, other optional biometric data records that are
-    available. Minimum accuracy requirements for the biometric verification are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On
+    applicant against biometric data records available on the PIV Card or in the PIV enrollment record.
+    Minimum accuracy requirements for the biometric verification are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On
```
Comment (gfiumara) [inline _FIPS201/requirements.md:334]:
```suggestion
    Minimum accuracy requirements for the biometric verification and presentation attack detection are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On
```

File: _FIPS201/requirements.md [hunk 3]
```diff
@@ -332,9 +330,8 @@ of the federal department or agency.
 + During the issuance process, the issuer **SHALL** verify that the individual to whom the PIV Card is to be
     issued is the same as the intended applicant/recipient as approved by the appropriate authority.
     Before the PIV Card is provided to the applicant, the issuer **SHALL** perform a one-to-one comparison of the
-    applicant against biometric data records available on the PIV Card or in the PIV enrollment record. The one-to-one
-    comparison requires either a comparison of fingerprints or, if unavailable, other optional biometric data records that are
-    available. Minimum accuracy requirements for the biometric verification are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On
+    applicant against biometric data records available on the PIV Card or in the PIV enrollment record.
+    Minimum accuracy requirements for the biometric verification and presentation attack detection are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On
```
Comment (gfiumara) [inline _FIPS201/requirements.md:334]:
```suggestion
    Minimum accuracy requirements for biometric verification and presentation attack detection are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On
```

**Pull request merged**